config/configChecker.go

```diff
@@ -66,8 +65,9 @@ func checkStakingV4MaxNodesChangeCfg(enableEpochsCfg EnableEpochs, numOfShards u
 			maxNodesConfigAdaptedForStakingV4 = true
 
 			if idx == 0 {
-				return fmt.Errorf("found config change in MaxNodesChangeEnableEpoch for StakingV4Step3EnableEpoch = %d, but %w ",
-					enableEpochsCfg.StakingV4Step3EnableEpoch, errNoMaxNodesConfigBeforeStakingV4)
+				log.Warn(fmt.Errorf("found config change in MaxNodesChangeEnableEpoch for StakingV4Step3EnableEpoch = %d, but %w ",
```

Comment:
this could have been:
```GO
log.Warn("found config change in MaxNodesChangeEnableEpoch for StakingV4Step3EnableEpoch", "StakingV4Step3EnableEpoch", enableEpochsCfg.StakingV4Step3EnableEpoch, "error", errNoMaxNodesConfigBeforeStakingV4)

```